### DISCUSSION TITLE
Please add bluetooth 5.1+(5.2, 5.3, 5.4.....) support on elementary, (i think it needs a kernel patch)

### DISCUSSION THREAD

**Discussion by takashialpha** [elementary/triage#529] [Ideas]
Please add bluetooth 5.1+(5.2, 5.3, 5.4.....) support on elementary

**Comment by danirabbit** [comment]
We use the Ubuntu LTS kernel and we’ve just updated to Linux 6.11 which as I understand has support for Bluetooth 5.3 for several chipsets. So closing as solved!
